test/docker_tests/test_docker_compose_scenarios.py

```
@@ -0,0 +1,440 @@
+    with open(compose_file, 'w') as f:
+        yaml.dump(compose_config, f)
+
+    # Run docker compose
+    result = _run_docker_compose(compose_file, "netalertx-host-net")
+
+    # Check that it doesn't fail with network-related errors
+    assert "not running with --network=host" not in result.output
+    # Container should start (may fail later for other reasons, but network should be OK)
+
+
+def test_host_network_compose() -> None:
```

Ruff (0.14.2)

Redefinition of unused `test_host_network_compose` from line 218

(F811)